addons/website_sale/views/variant_templates.xml

```diff
@@ -51,7 +55,7 @@
                                                 t-att-data-value_name="ptav.name"
                                                 t-att-data-attribute_name="ptav.attribute_id.name"
                                                 t-att-data-is_custom="ptav.is_custom"
-                                                t-att-data-is_single="single"
+                                                t-att-data-is_single="single if ptal.attribute_id.display_type == 'radio' else False"
```

Comment:
So when we create 'multi' attribute with only one variant, it will treat it a single and will not display in the popup when we add the quantity. But if we use 'multi' variant with multiple attributes, it will work fine. So this is basically done to handle this case. Here is a link for better understanding https://app.screencastify.com/v3/watch/XANG4ne0iKFU79LON6P9

Reply:
```suggestion
                                                t-att-data-is_single="single if ptal.attribute_id.display_type != 'multi' else False"
```
So shouldn't your condition be like this? :thinking: 
Because you need this single attribute for other `display_type` other then `multi`. No?

Reply:
Yes my bad. This would be more apt. :)